pytest.ini

```diff
@@ -1,7 +1,4 @@
 [pytest]
-# Fail if coverage falls below 95%
-# NOTE: If debug breakpoints aren't working, comment out the code coverage line below
-addopts = --cov samtranslator --cov-report term-missing --cov-fail-under 95
```

Comment:
Is the coverage in all the commands we need it to be on the makefile?

Reply:
Yes, right now we have only `make pr` which needs coverage and --cov is present in the Makefile already for `make pr`. Currently, we don't have 95% test coverage for integ tests. Removing these lines was necessary for `make e2e-tests` to pass